packages/node/README.md

```diff
@@ -34,15 +34,27 @@ npm install @imgly/background-removal-node
 
 ## Usage
 
-```typescript
-import {removeBackground} from "@imgly/background-removal-node";
-// const {removeBackground} = require("@imgly/background-removal-node");
-
-let image_src: ImageData | ArrayBuffer | Uint8Array | Blob | URL | string = ...;
+```js
+const { removeBackground } = require('@imgly/background-removal-node');
+const fs = require('fs');
+
+async function demoRemoveBackground(imageUrl) {
+  console.log('Removing background from: ' + imageUrl);
+  const blob = await removeBackground(imageUrl, { debug: false });
+  const buffer = await blob.arrayBuffer();
+  try {
+    await fs.promises.mkdir('tmp', { recursive: true });
+    await fs.promises.writeFile('tmp/output.png', Buffer.from(buffer));
+    console.log('Image saved to tmp/output.png');
```

Comment:
I am not sure about console outputs here

Reply:
Personally I find them helpful when nothing is visibly happening.